uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -297,20 +297,18 @@ <h4 class="modal-title">
                     <input
                       type="file"
                       nv-file-select
+                      ng-readonly="$ctrl.isMode('view')"
                       ng-disabled="$ctrl.isMode('view')"
```

Comment:
Is this `ng-disabled` required? It is already present in the parent
`<div class="chr_leave-request-files_upload" ng-disabled="$ctrl.isMode('view')">`